src/xrEngine/EngineAPI.h

```diff
@@ -83,9 +83,9 @@ class ENGINE_API CEngineAPI
 
 ENGINE_API bool is_enough_address_space_available();
 
-#define NEW_INSTANCE(a) Engine.External.pCreate(a)
+#define NEW_INSTANCE(a) Engine.External.pCreate ? Engine.External.pCreate(a) : nullptr
```

Comment:
It's better and more safe to assign pointer to a dummy function:
- create dummy will just return nullptr 
- destroy dummy will assert that passed pointer is nullptr.